robyn/scaffold/sqlmodel/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11-bookworm
+
+WORKDIR /workspace
+
+RUN pip install --no-cache-dir --upgrade -r requirements.txt
+
+COPY . .
```

Comment:
Hey @leogregianin 👋 

Thank you for the PR, great work! ✨ 

One suggestion - we will have to invert the order of lines 7 and 5.